package.json

```
@@ -112,6 +112,7 @@
 		"eslint-plugin-wpcalypso": "5.0.0",
 		"glob": "7.1.6",
 		"husky": "4.3.8",
+		"jetpack-cli": "file:tools/cli",
```

Instead of the postinstall script that does a yarn link, it looks like we can just change this line to
```suggestion
		"jetpack-cli": "link:tools/cli",
```
and it will do what we actually want here.

---

Hmm. Except for https://github.com/yarnpkg/yarn/issues/8290 apparently. Sigh.

---

I'm open to yarn v2 being able to handle this better, but that's a bit of a more detailed upgrade than something can do within the context of this PR or make a blocker for this.